.github/workflows/ci.yml

```
@@ -0,0 +1,77 @@
+name: Vex Core CI
+
+on:
+    push: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)